spp_event_data/models/registrant.py

```diff
@@ -0,0 +1,43 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+from datetime import date
+
+from odoo import _, fields, models
+
+
+class OpenSPPRegistrant(models.Model):
+    _inherit = "res.partner"
+
+    event_data_ids = fields.One2many("spp.event.data", "registrar", "Event Data IDs")
+
+    def _compute_active_event(self, model):
+        active_event = self.env["spp.event.data"].search(
+            [
+                ("registrar", "=", self.id),
+                ("model", "=", model),
+                ("expiry_date", "=", False),
```

Comment:
`expiry_date` is not `expired_date`. Also by default we ignore the expiry date, it s more for information for now.

Reply:
Fixed with [commit](https://github.com/OpenSPP/openspp-registry/pull/52/commits/01cb90811f52afc2ecb0251d3f4415bd316f9fb9)